core/tests/protractor_utils/action.js

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
Change this 2020! (I'll cover this in the commit addressing reviews from code owners!)